### ISSUE TITLE
Download / menu links don´t work on public share pages

### ISSUE THREAD

**Issue by splitt3r** [nextcloud/server#7509]
### Steps to reproduce
1. Share a file by link
2. Click on the download button in the upper right corner / the more icon

Another strange behaviour:
1. Share a pdf by link
2. Click on the download button in the upper right corner
3. Now you see a whitepage instead of the pdf

### Expected behaviour
Sth. should happen

### Actual behaviour
Nothing happens

## Server configuration detail

**Operating system:** Linux

**Webserver:** Apache (CentOS)

**Database:** sqlite3 3.7.17

**PHP version:** 7.1.12
Modules loaded: Core, date, libxml, openssl, pcre, zlib, filter, hash, Reflection, SPL, session, standard, cgi-fcgi, bcmath, bz2, calendar, ctype, curl, dom, mbstring, fileinfo, ftp, gd, gettext, iconv, imap, intl, json, exif, mcrypt, mysqlnd, PDO, pgsql, apcu, posix, shmop, SimpleXML, soap, sockets, sqlite3, sysvmsg, sysvsem, sysvshm, tidy, tokenizer, xml, xmlwriter, xsl, mysqli, pdo_mysql, pdo_pgsql, pdo_sqlite, wddx, xmlreader, apc, imagick, zip, Phar, Zend OPcache

**Nextcloud version:** 13.0.0 Beta 3

**Updated from an older Nextcloud/ownCloud or fresh install:** fresh

**Where did you install Nextcloud from:** official zip

<details><summary>Signing status</summary>

Integrity checker has been disabled. Integrity cannot be verified.
</details>

<details><summary>List of activated apps</summary>

```
Enabled:
 - comments: 1.3.0
 - dav: 1.4.5
 - federatedfilesharing: 1.3.1
 - federation: 1.3.0
 - files: 1.8.0
 - files_sharing: 1.5.0
 - files_trashbin: 1.3.0
 - files_versions: 1.6.0
 - issuetemplate: 0.3.0
 - lookup_server_connector: 1.1.0
 - oauth2: 1.1.0
 - provisioning_api: 1.3.0
 - sharebymail: 1.3.0
 - systemtags: 1.3.0
 - theming: 1.4.1
 - twofactor_backupcodes: 1.2.3
 - updatenotification: 1.3.0
 - workflowengine: 1.3.0
Disabled:
 - admin_audit
 - encryption
 - files_external
 - testing
 - user_ldap
```
</details>

<details><summary>Configuration (config/config.php)</summary>

```
{
    "instanceid": "***REMOVED SENSITIVE VALUE***",
    "passwordsalt": "***REMOVED SENSITIVE VALUE***",
    "secret": "***REMOVED SENSITIVE VALUE***",
    "trusted_domains": [
        "***REMOVED SENSITIVE VALUE***"
    ],
    "datadirectory": "***REMOVED SENSITIVE VALUE***",
    "overwrite.cli.url": "***REMOVED SENSITIVE VALUE***",
    "dbtype": "sqlite3",
    "version": "13.0.0.8",
    "installed": true
}
```
</details>

**Are you using external storage, if yes which one:** no

**Are you using encryption:** no

**Are you using an external user-backend, if yes which one:** no

## Client configuration

**Browser:** Mozilla Firefox / Edge / Chrome

![Download](https://user-images.githubusercontent.com/1869584/33985126-f521c324-e0b9-11e7-91bf-61e4c57a78ce.png)

The menu doesn´t show up because of

```
#header .menu {
    display: none;
}
```

which seems to override the value from the .open class.

**Comment by alberizo** [comment]
I have the same problem. When I share anything the link give error "The shared file doesnt exists",

The problem is the & in the url before "&download", if I change it to "/download" is working OK.

Nope
https://.......com/cloud/index.php/s/3OARtUk8WrZdp9V&download

Yes
https://.......com/cloud/index.php/s/3OARtUk8WrZdp9V/download

**Comment by splitt3r** [comment]
Hm i don´t know if you problem is related because if i remove the display: none; mentioned above the issue seems to be fixed. But the pdf problem is still there (pdfviewer is hidden after pressing the download / more button).

**Comment by alberizo** [comment]
You are right. I wanted open an issue about my problem, I saw yours and I thought What a coincidence! ... only 3 minutes ago. But I was wrong, isn't the same issue. I´m sorry

**Comment by MorrisJobke** [comment]
This is partly fixed by #7566

**Comment by MorrisJobke** [comment]
The menu in the header now properly opens as well as the PDF viewer is not hidden anymore. I will close this for now. Could you give this a test once the next beta is out? Thanks
